src/vscrollpanel.cpp

```diff
@@ -81,11 +81,24 @@ void VScrollPanel::draw(NVGcontext *ctx) {
     mChildPreferredHeight = child->preferredSize(ctx).y();
     float scrollh = height() *
         std::min(1.0f, height() / (float) mChildPreferredHeight);
+        
+    float offset = -mScroll*(mChildPreferredHeight - mSize.y());
 
     nvgSave(ctx);
     nvgTranslate(ctx, mPos.x(), mPos.y());
     nvgIntersectScissor(ctx, 0, 0, mSize.x(), mSize.y());
-    nvgTranslate(ctx, 0, -mScroll*(mChildPreferredHeight - mSize.y()));
+    nvgTranslate(ctx, 0, offset);
+
+
+    Widget *parentWindow = (Widget*)window();
```

Comment:
The loop applies the offset to all the popups that are a child of the current screen.
( This is because when you make a popup button, the popup get's attached to the screen since it's a window ).
This is also why the nvgTranslate doesn't work on the popup, hence the workaround using the anchor.